config.mk

```diff
@@ -108,15 +108,15 @@ makejobs=-j2
 # hard drive space.
 pass2_languages=c,c++,objc,obj-c++
 
-# Download protocol (https|ftp)
+# Download protocol (http|ftp)
 # Specify here the protocol you want to use for downloading the packages.
-download_protocol=https
+download_protocol=http
 
 # Force downloader (curl|wget)
 # You may specify here 'wget' or 'curl'. If this variable is empty or commented,
 # web downloader tool will be auto-detected in the following order: cURL, Wget.
 # You must have either Wget or cURL installed to use dc-chain.
-#force_downloader=curl
+force_downloader=wget
```

Comment:
This (`force_downloader`) should be commented by default, to let the system determines itself the available downloader. It will start from cURL (by requested by some users on our [Discord](https://discord.com/channels/488332893324836864/614912396460556298)) then fallback to Wget

Reply:
The problem seems to be related to the one using `https`  to `curl` but not `wget` getting the `config.guess` and `http` only URL